pkg/move/atomic.go

```diff
@@ -27,20 +26,20 @@ func Atomic(work string, copyFunc CopyFunc) CopyFunc {
 
 		defer func() {
 			if err != nil {
-				if cleanupErr := fs.RemoveAll(work); cleanupErr != nil {
+				if cleanupErr := os.RemoveAll(work); cleanupErr != nil {
 					log.Error(err, "failed cleanup of workdir after failure")
 				}
```

Comment:
this should be `log.Error(cleanupErr, ...)` no? an alternative would be to use `errors.Join`